src/views/registration-js/attendees/content.php

```diff
@@ -68,5 +68,6 @@
 				?>
 			<?php endwhile; ?>
 		</script>
+		<?php Tribe__Tickets_Plus__Meta__Field__Birth::output_javascript() ?>
```

Comment:
:no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: 5dd5a6a686fb891d6f860eb451d2db96</sub>

Reply:
Just add a do_action() here and hook into that.

Reply:
This view also needs a since/version ping